aries-site/src/examples/components/maskedinput/MaskedValidationExample.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Box, Button, FormField, Form, MaskedInput } from 'grommet';
+
+const IPv4ElementExp = /^[0-1][0-9][0-9]$|^2[0-4][0-9]$|^25[0-5]$|^[0-9][0-9]$|^[0-9]$/;
+
+export const MaskedValidationExample = () => {
+  return (
+    <Form>
+      <FormField
+        label="IP Address"
+        name="maskedIpAddressValidation"
+        htmlFor="masked-ip-address-validation"
+        validate={{
```

Comment:
This one is less dense than the mask logic, but consider moving to its own variable.